worker/src/RTC/Transport.cpp

```diff
@@ -2340,6 +2340,26 @@ namespace RTC
 		}
 	}
 
+	static inline uint32_t IncreaseConsumerLayer(
```

Comment:
Can we please not move this into a separate method? I don't see any need. We never mix class method with static methods together.